src/components/AppHelp.vue

```diff
@@ -0,0 +1,255 @@
+						{{ t('notes', 'Android app: Nextcloud Notes by Niedermann IT-Dienstleistungen ') }}
+					</a>
+					<div>
+						<div class="badge">
+							<a target="_blank" href="https://play.google.com/store/apps/details?id=it.niedermann.owncloud.notes">
+								<img :src="getRoute('badge_playstore.svg')" class="appstore-badge badge-playstore-fix">
+							</a>
+						</div>
+						<div class="badge">
+							<a target="_blank" href="https://f-droid.org/repository/browse/?fdid=it.niedermann.owncloud.notes">
+								<img :src="getRoute('badge_fdroid.svg')" class="appstore-badge">
+							</a>
+						</div>
+					</div>
+				</div>
+				<div class="badge-wrapper">
+					<a href="https://github.com/phedlund/CloudNotes">
+						{{ t('notes', 'iOS app: CloudNotes - Nextcloud Notes by Peter Hedlund') }}
+					</a>
+					<div>
+						<div class="badge">
+							<a target="_blank" href="https://apps.apple.com/app/cloudnotes-owncloud-notes/id813973264">
+								<img :src="getRoute('badge_applestore.svg')" class="appstore-badge badge-playstore-fix">
+							</a>
+						</div>
+					</div>
+				</div>
+			</AppSettingsSection>
+		</AppSettingsDialog>
+	</div>
+</template>
+
+<script>
+
+import {
+	AppSettingsDialog,
+	AppSettingsSection,
+} from '@nextcloud/vue'
+import { createNote } from '../NotesService'
+import { getDefaultSampleNote } from '../Util'
+import { generateFilePath } from '@nextcloud/router'
+
+export default {
+	name: 'AppHelp',
+
+	components: {
+		AppSettingsDialog,
+			AppSettingsSection,
+	},
+
+	props: {
+		settingsOpen: Boolean,
+	},
+
+	data() {
+		return {}
+	},
+
+	computed: {
+		getShortcuts() {
+			return [
+				{ shortcut: t('notes', 'CTRL+\''), action: t('notes', 'Make the selection a quote') },
+				{ shortcut: t('notes', 'CTRL+B'), action: t('notes', 'Make the selection bold') },
+				{ shortcut: t('notes', 'CTRL+E'), action: t('notes', 'cleanBlock') },
+				{ shortcut: t('notes', 'CTRL+H'), action: t('notes', 'toggleHeadingSmaller') },
+				{ shortcut: t('notes', 'CTRL+I'), action: t('notes', 'Make the selection italic') },
+				{ shortcut: t('notes', 'CTRL+K'), action: t('notes', 'Insert link at cursor') },
+				{ shortcut: t('notes', 'CTRL+L'), action: t('notes', 'Makes the current line a list element') },
+				{ shortcut: t('notes', 'CTRL+P'), action: t('notes', 'Toggle between preview') },
+				{ shortcut: t('notes', 'CTRL+Alt+C'), action: t('notes', 'The selection will be turned into monospace') },
+				{ shortcut: t('notes', 'CTRL+Alt+I'), action: t('notes', 'Insert image at the cursor') },
+				{ shortcut: t('notes', 'CTRL+Alt+L'), action: t('notes', 'kes the current line a list element with a number') },
+				{ shortcut: t('notes', 'SHIFT+CTRL+H'), action: t('notes', 'toggleHeadingBigger') },
+				{ shortcut: t('notes', 'F9'), action: t('notes', 'toggleSideBySide') },
+				{ shortcut: t('notes', 'F11'), action: t('notes', 'Make the note fullscreen') },
```

Comment:
Hmm. This is not the same like the fullscreen action in the three-dots menu. Hence, remove this entry?

Reply:
But it is? In the end the tree dot menu calls "requestFullscreen()" on the dom, which acts the exact same way as F11.

Reply:
At least not in my Firefox: pressing F11 goes into fullscreen but all the Nextcloud frame and app navigation is still visible. Using the fullscreen option shows the note only without any distracting frame.